src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/TaintedDataAnalysis/SqlSinks.cs

```diff
@@ -48,6 +48,16 @@ static SqlSinks()
                 },
                 sinkMethodParameters: null);
 
+            sinkInfosBuilder.AddSinkInfo(
```

Comment:
Could you fill an issue in vsdocs to mention this new sink?

Reply:
I tried to search, but failed to find the repository.

Reply:
Looks like there is no mention of the sinks in the [doc](https://docs.microsoft.com/visualstudio/code-quality/ca3001) so let's do nothing. Sorry for the useless comment :)

Reply:
Not something to address in this PR, but if there's interest in documenting all the sources / sanitizers / sinks, might be neat to have a tool to generate Markdown that could be copied and pasted into the docs.